.github/CODEOWNERS

```diff
@@ -38,7 +38,7 @@
 # TODO(#10619): Add @bansalnitish as owner after 2020-10-31
 /core/templates/components/ @srijanreddy98
 /core/templates/domain/ @kevintab95
-/core/templates/pages/ @seanlip
+/core/templates/pages/ @srijanreddy98
```

Comment:
Hi @srijanreddy98 -- I think it might make sense for you to look after this going forward, since it intersects with the Angular migration quite a bit. What do you think? (Normally, specific pages will be owned by more specific codeowners listed below -- this part at the top is just to make sure that every file has an owner.)

Reply:
SG!